docs/protocol.html

```diff
@@ -16,8 +16,11 @@
 -->
 
 <!--#include virtual="../includes/header.html" -->
-
-<h3><a id="protocol" href="#protocol">Kafka Wire Protocol</a></h3>
+<!--#include virtual="../includes/top.html" -->
+<div class="content">
+    <!--#include virtual="../includes/nav.html" -->
+    <div class="right">
```

Comment:
Forgive my poor html knowledge, but do these `div` blocks need to end somewhere?